hardware_interface/include/hardware_interface/sensor_interface.hpp

```diff
@@ -29,7 +29,7 @@ namespace hardware_interface
 {
 
 /**
-  * \brief Virtual Class to implement when integrating a stand-alone sensor into ros2_control.
+  * /// Virtual Class to implement when integrating a stand-alone sensor into ros2_control.
```

Comment:
please extract the brief line as a first line of the doc block

Reply:
I changed to `///` all single line doc sentences.